src/services/lease-service/rental-objects.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Self-contained service, ready to be extracted into a microservice if appropriate.
+ *
+ * All adapters such as database clients etc. should go into subfolders of the service,
+ * not in a general top-level adapter folder to avoid service interdependencies (but of
+ * course, there are always exceptions).
+ */
+import KoaRouter from '@koa/router'
+import { generateRouteMetadata } from 'onecore-utilities'
+
+import * as leasingAdapter from '../../adapters/leasing-adapter'
+
+export const routes = (router: KoaRouter) => {
+  /**
+   * @swagger
+   * /vacant-parkingspaces:
+   *   get:
+   *     summary: Get all vacant parking spaces
+   *     tags:
+   *       - Lease service
+   *     description: Retrieves a list of all vacant parking spaces.
+   *     responses:
+   *       '200':
+   *         description: A list of vacant parking spaces.
+   *         content:
+   *           application/json:
+   *             schema:
+   *               type: object
+   *               properties:
+   *                 content:
+   *                   type: array
+   *                   items:
+   *                     $ref: '#/components/schemas/RentalObject'
```

Comment:
I'm a  bit confused about this $ref usage. We use this in property-base and work-order projects and I don't think it's set up here in leasing (or in property-management for that matter). Do you know anything about this?